appium/tests/specs/mock/inbox/ReadTextEmail.spec.ts

```diff
@@ -32,14 +32,19 @@ describe('INBOX: ', () => {
     mockApi.ekmConfig = {
       returnKeys: [ekmKeySamples.e2eValidKey.prv]
     }
+    mockApi.googleConfig = {
+      accounts: {
+        'e2e.enterprise.test@flowcrypt.com': {
```

Comment:
Better if we define types for mock gmail accounts like below.
This way, we can benefit from auto complete and developer will not make mistake with typo.

```
type GmailMockAccountType = 'e2e.enterprise.test@flowcrypt.com` | 'anymockaccount@test.com'....;
export type GoogleConfig = {
  allowedRecipients: string[]
  accounts: { [email: GmailMockAccountType]: GoogleMockAccount }
};

```

Reply:
I disagree. I think there is benefit in having the email addresses directly visible.

I'd also start to make test values directly visible but that would be for another issue. It makes tests harder to read when everything is hidden behind a variable.

If developer makes a typo, the test will stop working.

Reply:
Sorry but I think you misunderstood above.
Even if we define types above, below code won't change. It indeed displays email address directly this way.
It just enables auto-completion and shows error if there's a typo.

    mockApi.googleConfig = {
      accounts: {
        'e2e.enterprise.test@flowcrypt.com': {

By the way, I found that above code I posted shows error. I updated it like follows.

```
type GmailMockAccountType = 'e2e.enterprise.test@flowcrypt.com' | 'anymockaccount@test.com';
export type GoogleConfig = {
  accounts: Partial<Record<GmailMockAccountType, GoogleMockAccount>>
}
```

![image](https://user-images.githubusercontent.com/77344191/174239071-40bd5b5a-35eb-4c07-a7e9-bed781f64a60.png)

Reply:
Yup - I later noticed I misunderstood. It's a great suggestion.

Reply:
Nice improvement for code readability, will implement it 👍